about.html

```diff
@@ -0,0 +1,100 @@
+           
+                <span>Thank you for taking the time to learn about us. We're here to make your shopping experience delightful and memorable. If you have any questions, need assistance, or simply want to share your feedback, please don't hesitate to contact our friendly team. <br> <a href="">lawmsangivarte22@gmail.com</a></span>
+                
+            </div>
+        </section>
+
+        <section class="contact">
+            <div class="contact-container">
+                <h2>Unleash your creativity and share your own designs with us - join the magic of personalization!</h2>
+                <div class="line"></div>
+                <p>To proceed with sharing your own designs, simply reach out to our dedicated design team through our contact form or email. We'll work closely with you to bring your vision to life, ensuring every detail is perfected. Join us in creating unique and unforgettable personalized gifts!</p>
+                <img src="images/9.jpeg" alt="img">
+            </div>
+
+            <div class="contact-container">
+                <h2>Embrace the opportunity to monetize your artistic flair and captivate a wider audience by selling your own uniquely crafted designs with us.</h2>
+                <div class="line"></div>
+                <p>To embark on your journey of selling your own designs, simply submit your portfolio and design samples to our dedicated team via our online submission form. We'll carefully review your work and get in touch with you to discuss collaboration opportunities. Let your creative vision thrive and turn your passion into a flourishing business with our support and platform.</p>
+                <div class="image-container">
+                    <div class="contact-image">
+                        <img src="images/7.jpeg" alt="shirt">
+                    </div>
+                    <div class="contact-image1">
+                        <img src="images/8.jpeg" alt="shirt">
+                    </div>
+                </div>
+            </div>
+        </section>
+
+         <section class="section-partner">
+        <div class="partner-container">
+            <div class="partner-header">
+                    <h2>Partner</h2>
+                    <div class="partner-line"></div>
+            </div>
+            <div class="partner-content">
+                <img src="images/amazon-64.png" alt="amazon">
+                <img src="images/nike-2-64.png" alt="nike">
+                <img src="images/apple-64.png" alt="apple">
+                <img src="images/google-plus-3-64.png" alt="google">
+                <img src="images/facebook-2-64.png" alt="fb">
+            </div>
+        </div>
```

Comment:
You did well implementing some of the changes mentioned by the previous reviewer 😄 but there seems to be just a minor issue to be addressed. For the partner's section of the `about page`, as suggested by the previous reviewer, you should try to make it similar to the design given by using the right background color for this section , it is always good to replicate a design as much as possible for a great UI👍

Behance design : 
![image](https://github.com/Lawmsangi/Module-One-Capstone/assets/20537712/42745087-335c-46ee-b164-62a28a1315f9)

Your design :
![image](https://github.com/Lawmsangi/Module-One-Capstone/assets/20537712/c44d2b55-206e-4ee5-9491-87cf594e2be7)